TableView/TextField.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  TextField.swift
+//  TableView
+//
+//  Created by Oleg Yankiwskyi on 4/27/18.
+//  Copyright © 2018 Oleg Yankiwskyi. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class TextField: UITextField, UITextFieldDelegate {
+    
+    var callAction: ( (String)->() )?
+    
+    init(title: String, action: ((String)->())? ) {
```

Comment:
UITextField doesn't contain `title` property; they're either `text` or `placeholder`.
Use proper naming to avoid confusion